amoro-ams/amoro-ams-server/src/main/java/org/apache/amoro/server/dashboard/PaimonTableDescriptor.java

```diff
@@ -525,14 +525,23 @@ public List<TagOrBranchInfo> getTableBranches(AmoroTable<?> amoroTable) {
   @Override
   public List<ConsumerInfo> getTableConsumerInfos(AmoroTable<?> amoroTable) {
     FileStoreTable table = getTable(amoroTable);
+    FileStore<?> store = table.store();
     ConsumerManager consumerManager = new ConsumerManager(table.fileIO(), table.location());
     List<ConsumerInfo> consumerInfos = new ArrayList<>();
     try {
       consumerManager
           .consumers()
           .forEach(
-              (consumerId, nextSnapshot) ->
-                  consumerInfos.add(new ConsumerInfo(consumerId, nextSnapshot)));
+              (consumerId, nextSnapshotId) -> {
+                long currentSnapshotId = nextSnapshotId - 1;
```

Comment:
If can ensure that nextSnapshotId >0, i think don’t need this judgment.

Reply:
if first snapshot id from flink checkpoint is 1, we need judgment nextSnapshotId > 1,because of get consumer is nextSnapshotId

![image](https://github.com/user-attachments/assets/fbd52799-4c79-4a44-83bb-0a93802f2e71)

Reply:
If paimon itself can already determine that this is >1, do we not need to judge anymore? If we add judgment, if there is an exception, the stack info will be lost.

Reply:
![image](https://github.com/user-attachments/assets/3693c54b-1179-410f-b309-0ae0f24127b7)

Reply:
I read this part of the read code, the underlying exception catch, but did not add the SnapshotId judgment range